client/title_card.js

```diff
@@ -64,19 +68,22 @@ limitations under the License.
 import asset from '/client/asset/asset.js';
 
 function makeEmptyTitleCard() {
-  var elem = document.createElement('div');
+  const elem = document.createElement('div');
   elem.id = 'title-card';
   return elem;
 }
 
 function makeDefaultTitleCard(config) {
-  var elem = makeEmptyTitleCard();
-  if (typeof config.title === 'string') {
+  const elem = makeEmptyTitleCard();
+  if (config.path) {
```

Comment:
config.image (per the comments above?).
Also consider renaming the argument config->credit, since it's easier to follow.

Reply:
Done!